internal/db/db.go

```diff
@@ -161,7 +161,8 @@ func (tx *holdTx) Insert(rec Record) error {
 
 func (tx *holdTx) Update(oldRec, newRec Record) error {
 	tx.ensureWrite()
-	//only necessary if you change the key
+	//Delete is only necessary if you change the recordID
```

Comment:
I only just noticed this—sorry. I'd like it to be invalid to change the primary key of a row via update. I think doing so would break some assumptions that we'll find helpful down the line.